Ginger/Ginger/AutomatePageLib/NewAutomatePage.xaml.cs

```diff
@@ -715,7 +755,7 @@ private async void App_AutomateBusinessFlowEventAsync(AutomateEventArgs args)
             switch (args.EventType)
             {
                 case AutomateEventArgs.eEventType.Automate:
-                    LoadBusinessFlowToAutomate((BusinessFlow)args.Object);
+                    await LoadBusinessFlowToAutomate((BusinessFlow)args.Object);
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)